.github/workflows/test.yml

```diff
@@ -0,0 +1,56 @@
+name: Deriv-App
+on:
+  pull_request:
+    branches:
+    - master
+env:
+  CACHE_VERSION: ${{ github.sha }}
+  COVERALLS_REPO_TOKEN: ${{ secrets.COVERALLS_REPO_TOKEN }}
+  DATADOG_CLIENT_TOKEN_LOGS: ${{ secrets.DATADOG_CLIENT_TOKEN_LOGS }}
+  DATADOG_SESSION_REPLAY_SAMPLE_RATE: ${{ secrets.DATADOG_SESSION_REPLAY_SAMPLE_RATE }}
+  DATADOG_SESSION_SAMPLE_RATE: ${{ secrets.DATADOG_SESSION_SAMPLE_RATE }}
+  DATADOG_SESSION_SAMPLE_RATE_LOGS: ${{ secrets.DATADOG_SESSION_SAMPLE_RATE_LOGS }}
+  GD_API_KEY: ${{ secrets.GD_API_KEY }}
+  GD_APP_ID: ${{ secrets.GD_APP_ID }}
+  GD_CLIENT_ID: ${{ secrets.GD_CLIENT_ID }}
+  RUDDERSTACK_PRODUCTION_KEY: ${{ secrets.RUDDERSTACK_PRODUCTION_KEY }}
+  RUDDERSTACK_STAGING_KEY: ${{ secrets.RUDDERSTACK_STAGING_KEY }}
+  RUDDERSTACK_URL: ${{ secrets.RUDDERSTACK_URL }}
+  SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
```

Comment:
Secrets won't be passed to the workflow as it runs on pull_requests - https://securitylab.github.com/research/github-actions-preventing-pwn-requests/

Reply:
We should have two different GitHub actions as mentioned in https://securitylab.github.com/research/github-actions-preventing-pwn-requests/

Reply:
i need to categorize these before merging it.
Some need to go to environment variables. like datadog and rudderstack keys
and some need to removed from here, like slack webhook url which we dont need them here.
Anyways with the new github configuration that we did last week, no one can have access to repo secrets from forked repositories/Pull requests.

Reply:
@balakrishna-deriv I removed all of the env variables and secrets for test workflow, please take a look at it one more time.